faq-bundle/src/Resources/contao/dca/tl_faq.php

```diff
@@ -598,7 +599,7 @@ public function getSerpUrl(FaqModel $objFaq)
 
 		if (!$objTarget = PageModel::findByPk($jumpTo))
 		{
-			throw new Exception('Invalid jumpTo page: ' . $jumpTo);
+			return StringUtil::ampersand(Environment::get('request'));
```

Comment:
Is `Environment::get('request')` correct here? I think this is deprecated in Contao 5.0.

Reply:
Oh, the events seem to do the same currently so this is consistent.

Reply:
Yes, but we are still using it a lot in Contao 5 and especially in the two other methods that generate URLs:

https://github.com/contao/contao/blob/4586b2f5b58075e5cfd076e1913cbf86dfd4337e/calendar-bundle/src/Resources/contao/classes/Events.php#L465-L468

https://github.com/contao/contao/blob/4586b2f5b58075e5cfd076e1913cbf86dfd4337e/news-bundle/src/Resources/contao/classes/News.php#L446-L449